argo-cd-bridges/gitlab/.helm/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: gitlab-argo-bridge
+description: A Helm chart which builds and deploys a CronJob that syncs GitLab projects (in a group) with an ArgoCD instance
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+appVersion: 1.16.0
```

Comment:
How much do we care about this? Would we create a new release of this repo every time we update this app for example? Or will this app have it's own release schedule and versioning? If the latter I would suggest we start with either `0.1.0` or `1.0.0`.

Reply:
I don't think we do... This is just what helm initialized here. I will change it.